components/Chips/examples/ChipsTypicalUseViewController.m

```diff
@@ -29,6 +30,7 @@ - (instancetype)init {
   if (self) {
     _sizingChip = [[MDCChipView alloc] init];
     _sizingChip.mdc_adjustsFontForContentSizeCategory = YES;
+    self.shapeScheme = [[MDCShapeScheme alloc] init];
```

Comment:
_Tip:_ I thought within an `init` we were supposed to use an `ivar`, I may be wrong here?